software/cookcli.yml

```
@@ -0,0 +1,11 @@
+name: CookCLI
+website_url: https://cooklang.org
+source_code_url: https://github.com/cooklang/CookCLI
+description: Command-line tool for automating meal planning and shopping with Cooklang recipes, scriptable for UNIX workflows, includes web server.
+licenses:
+  - MIT
+platforms:
+  - Rust
+  - Docker
```

There seem no install instructions in https://github.com/cooklang/CookCLI?tab=readme-ov-file#-installation for docker
```suggestion
platforms:
  - Rust
```